tests/bugs/test_bytecode.py

```diff
@@ -0,0 +1,52 @@
+"""
+A bytecode sequence that caused bug, calling to undefined opcode with filled stack
+"""
+
+import pytest
+
+from ethereum_test_tools import Account, Alloc, Environment, StateTestFiller, Transaction
+from ethereum_test_tools.vm.opcode import Opcode
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+REFERENCE_SPEC_GIT_PATH = "N/A"
+REFERENCE_SPEC_VERSION = "N/A"
+
+
+@pytest.mark.valid_from("Frontier")
+def test_bytecode(
+    state_test: StateTestFiller,
+    pre: Alloc,
+):
+    """
+    Original bytecode: 0x67FFFFFFFFFFFFFFFF600160006000FB
+    Original test: src/GeneralStateTestsFiller/stBugs/evmBytecodeFiller.json
+    """
+    code_worked = 1
+    code_contract = pre.deploy_contract(
+        balance=0,
+        code=Op.PUSH8(0xFFFFFFFFFFFFFFFF)
+        + Op.PUSH1(0x01)
+        + Op.PUSH1(0x00)
+        + Op.PUSH1(0x00)
+        + Opcode(0xFB),
+        storage={},
+    )
+
+    caller = pre.deploy_contract(
+        code=Op.SSTORE(0, Op.CALL(0, code_contract, 0, 0, 0, 0, 0)) + Op.SSTORE(code_worked, 1)
```

Comment:
```suggestion
        code=Op.SSTORE(0, Op.CALL(address=code_contract)) + Op.SSTORE(code_worked, 1)
```

Reply:
Oh cool, because there is actually a mistake here. Call passed 0 gas.